whylog/tests/tests_config/test_files/rules.yaml

```diff
@@ -0,0 +1,22 @@
+---
+causes: [3644641c-1404-4afd-a6bc-03a1a0dd6fbb, b7576241-643a-4285-b628-f2bcf72e43d0]
+constraints:
+- clues:
```

Comment:
It is a list of "clues", but really it identifies groups in those clues, not the clues themselves. `clue_groups`?